### ISSUE TITLE
Possable stocastic issue with AtomMapping during test_run_point_mutation_propose test

### ISSUE THREAD

**Issue by mikemhenry** [choderalab/perses#870]
It looks like once in a while CI will fail with this error:

https://github.com/choderalab/perses/runs/3634354459?check_suite_focus=true#step:9:303

I will use this issue to track errors like this, but as of now this is only in the investigation/collecting more data phase.

**Comment by ijpulidos** [comment]
Yeah, I have been hitting these stochastic failures as well. Just FYI.
https://github.com/choderalab/perses/pull/872/checks?check_run_id=3726693358#step:9:303

**Comment by ijpulidos** [comment]
I think we haven't hit this one in a while. Closing.
